index.rst

```diff
@@ -1,42 +1,198 @@
-.. _explicit-namespaces:
+.. explicit_namespaces documentation master file
 
-Explicit namespaces in import/export
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Explicit namespaces in import/export lists
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. extension:: ExplicitNamespaces
-    :shortdesc: Enable using the keyword ``type`` to specify the namespace of
-        entries in imports and exports (:ref:`explicit-namespaces`).
-        Implied by :extension:`TypeOperators` and :extension:`TypeFamilies`.
+    :shortdesc: Enable the use of the keywords ``type`` and ``data`` to specify the precise
+        entities exported from a module and brought into scope by an import declaration 
+        (:ref:`explicit-namespaces`).
 
+    :implied by: :extension:`TypeOperators`, :extension:`TypeFamilies`
     :since: 7.6.1
```

Comment:
Should we mention in the `:since:` field that `data` and namespace-specified imports were added in a later release?

Reply:
That is a good question &mdash; as I didn't know how to encode this information I left it.